src/js/charts/line.js

```diff
@@ -4,6 +4,8 @@
   function mg_line_color_text(elem, d, args) {
     elem.classed('mg-hover-line' + d.line_id + '-color', args.colors === null)
       .attr('fill', args.colors === null ? '' : args.colors[d.line_id - 1]);
+        elem.classed('mg-hover-line' + '-color', args.colors === null)
+          .attr('fill', args.colors === null ? 'black' : args.colors[d.line_id - 1]);
```

Comment:
The logic to make this black or not should not be conditional on the content of `args`. We should always apply this style regardless. 

Logically, this statement should be before the custom style is applied. It should be indented 2 spaces, not 4.

Also there is no need to add two strings together when they are constant types. I.e.: `'mg-hover-line' + '-color'` -> `'mg-hover-line-color'`.